exercises/practice/sum-of-multiples/tests/sum_of_multiples.cairo

```diff
@@ -0,0 +1,176 @@
+use sum_of_multiples::{sum_of_multiples as verify_sum_of_multiples};
+
+#[test]
+fn no_multiples_within_limit() {
+    let mut factors: Array<u32> = ArrayTrait::new();
+    factors.append(3);
+    factors.append(5);
+    let limit = 1;
+    let output = verify_sum_of_multiples(limit, factors);
+    assert(output == 0, 'Test failed: Expected 0.');
```

Comment:
```suggestion
    assert(output == 0, 'Test failed: Expected 0.');
    assert_eq!(output, 0);
```

Generally, this track uses `assert_eq!` when comparing output to expected values and `assert` when comparing output to Boolean values. We should probably stay consistent with the other exercises' test suites.